src/Codec/FFmpeg/Encode.hsc

```diff
@@ -753,29 +767,29 @@ avWriter outputFormat sp fname = do
                   modifyIORef lastPts (const newPts)
                   modifyIORef frameNum (+ fromIntegral numSamples)
 
-                  runWithError "Error encoding audio"
+                  _ <- runWithError "Error encoding audio"
                       (avcodec_send_frame ctx frame)
                   read_pkts
             return writeAudioFrame
           else
             return $ \_ -> return ()
 
-  videoWriter <- case mVideoStream of
+  vWriter <- case mVideoStream of
```

Comment:
Any reason for the name changes here?

Reply:
Name clashing with the videoWriter function so the change to reduce warnings